apps/web/src/pages/_app.tsx

```diff
@@ -28,6 +29,7 @@ function MyApp({ Component, pageProps }: AppProps) {
               <main className="m-auto max-w-3xl px-4 py-8">
                 <Component {...pageProps} />
               </main>
+              <Footer className="absolute bottom-0 right-[calc(50%_-_16rem)] mb-3"/>
```

Comment:
i wonder if we should have it a bit more low-key in the sidebar instead like we have it in the task? Then it won't be so distracting from the editing experience

I've updated the design in Figma, it's like this now
<img width="450" alt="image" src="https://user-images.githubusercontent.com/25622412/232333504-d0558ad3-a967-43e3-926f-374f9d7f5a21.png">

Reply:
Makes sense to be more lowkey for sure!